### ISSUE TITLE
Adding README and Contribution Guidelines

### ISSUE THREAD

**Issue by robinduerhager** [increpare/bfxr2#21]
Hey, i just saw your old bfxr Project and thought: This would be a great fit for a WebApp when i immediately saw, that you are actually working on one 😄 .

Are you accepting PRs, Issues / Issue Comments, etc, or is this Repo more private and right now just for testing and playing around?

If it already is open for contributions, i would recommend to add a README (maybe after the real initial structure of the project has been found -> React, Vue, Svelte, VanillaJS, Electron or whatever Stack you want to use) and some contribution guidelines, editorconfig, prettier configuration for formatting, etc.

**Comment by increpare** [comment]
Hey, yeah thanks for your comments. It's super early on still (right now stagnating a bit), not really fit for third-party contributions just now 😅

**Comment by Novalis15** [comment]
Yeah ofcourse, i'll wait then haha 😄. Hope my comments still help you in your decisions :).

**Comment by increpare** [comment]
readme + development added - contribution guidelines would be overkill IMO - people can make issues / report bugs as they like
